packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1630,6 +1630,24 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     public async generateSummary(options: IGenerateSummaryOptions): Promise<GenerateSummaryData | undefined> {
         const { fullTree, refreshLatestAck, summaryLogger } = options;
 
+        if (refreshLatestAck) {
+            const latestSummaryRefSeq = await this.refreshLatestSummaryAckFromServer(
+                ChildLogger.create(summaryLogger, undefined, { all: { safeSummary: true } }));
+
+            if (latestSummaryRefSeq > this.deltaManager.lastSequenceNumber) {
+                // We need to catch up to the latest summary's reference sequence number before pausing.
+                await PerformanceEvent.timedExecAsync(
```

Comment:
Is the purpose of this event to know how long it took to get current, or record that we actually were behind?
I feel like it should be rare event, and also it's possible that it's a bug and we will hang here forever, so maybe just having an event that we started waiting is better?

Reply:
I was thinking the duration would be useful too, what's the reason not to use perf event here? Just an extra event rarely?

Reply:
If we are never getting start events back, I can change it, otherwise I feel like both are useful.

Reply:
Forced start events